packages/rust-dprint-plugin-typescript/Cargo.toml

```diff
@@ -9,13 +9,23 @@ license = "MIT"
 homepage = "https://github.com/dsherret/dprint/tree/master/packages/rust-dprint-plugin-typescript"
 repository = "https://github.com/dsherret/dprint"
 
+[lib]
+name = "dprint_plugin_typescript"
+path = "src/lib.rs"
+
+[[bin]]
+name = "dprint_plugin_typescript"
```

Comment:
Instead of putting this in `rust-dprint-plugin-typescript` folder, could we do it in a new `rust-dprint` folder instead then have `dprint-plugin-typescript` as a dependency? I have the `dprint` crate and it would be good to make use of that.